src/parsers/manifestjson.js

```diff
@@ -136,7 +150,12 @@ export default class ManifestJSONParser extends JSONParser {
   _validate() {
     // Not all messages returned by the schema are fatal to Firefox, messages
     // that are just warnings should be added to this array.
-    const warnings = [messages.MANIFEST_PERMISSIONS.code];
+    const warnings = [
+      messages.MANIFEST_PERMISSIONS.code,
+      // Remove the following once the LWT aliases deprecated property should
+      // become errors on submission.
+      messages.MANIFEST_THEME_LWT_ALIAS.code,
```

Comment:
Once we are ready to mark these LWT aliases as errors instead of warnings, on the linter side we just need to remove this line (and so I'm going to file a follow up issue).

Reply:
Mind adding it to the comment if filed?